source/agora/common/DNS.d

```diff
@@ -616,6 +617,15 @@ public struct ResourceRecord
         return ResourceRecord(name, TYPE.A, CLASS.IN, ttl, RDATA(ipv4));
     }
 
+    /// Make a record of URI type
+    public static ResourceRecord make (TYPE type : TYPE.URI) (
+        Domain name, uint ttl, Address uri) @safe
+    {
+        return ResourceRecord(name, TYPE.URI, CLASS.IN, ttl, RDATA(URIRDATA(
+            0, 0, uri
```

Comment:
Those two parameters should be parameters to `make` (with sensible default values).